BaikalStandalone/Application/cl_render.cpp

```diff
@@ -65,6 +65,8 @@ namespace Baikal
 #ifdef ENABLE_DENOISER
         AddPostEffect(m_primary, PostEffectType::kMLDenoiser);
         m_output_accessor = std::make_unique<RendererOutputAccessor>("images", m_width, m_height);
+        m_post_effect->SetParameter("gpu_memory_fraction", settings.gpu_mem_fraction);
```

Comment:
add TODO: it's applicable only for MLDenoiser

Reply:
Done